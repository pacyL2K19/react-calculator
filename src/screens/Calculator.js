import { Box } from '@material-ui/core';
import App from '../components/App';

const Calculator = () => (
  <Box width="370px" mt={5} m="auto"><App /></Box>
);

export default Calculator;
