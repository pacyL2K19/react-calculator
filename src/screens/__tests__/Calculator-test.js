import React from 'react';
import { render } from '@testing-library/react';
import Calculator from '../Calculator';

describe('Calculator page', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<Calculator />);
    expect(asFragment(<Calculator />)).toMatchSnapshot();
  });
});
