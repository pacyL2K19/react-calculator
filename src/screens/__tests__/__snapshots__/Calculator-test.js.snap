// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator page should match the snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div>
    <div
      className="element text"
    >
      0
    </div>
    <div>
      <div>
        <div
          className="button btn-primary"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            AC
          </span>
        </div>
        <div
          className="button btn-primary"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            +/-
          </span>
        </div>
        <div
          className="button btn-primary"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            %
          </span>
        </div>
        <div
          className="button btn-primary"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            ÷
          </span>
        </div>
      </div>
      <div>
        <div
          className="button"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            7
          </span>
        </div>
        <div
          className="button"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            8
          </span>
        </div>
        <div
          className="button"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            9
          </span>
        </div>
        <div
          className="button btn-primary"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            X
          </span>
        </div>
      </div>
      <div>
        <div
          className="button"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            4
          </span>
        </div>
        <div
          className="button"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            5
          </span>
        </div>
        <div
          className="button"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            6
          </span>
        </div>
        <div
          className="button btn-primary"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            −
          </span>
        </div>
      </div>
      <div>
        <div
          className="button"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            1
          </span>
        </div>
        <div
          className="button"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            2
          </span>
        </div>
        <div
          className="button"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            3
          </span>
        </div>
        <div
          className="button btn-primary"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            +
          </span>
        </div>
      </div>
      <div>
        <div
          className="button button-zero"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            0
          </span>
        </div>
        <div
          className="button"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            .
          </span>
        </div>
        <div
          className="button btn-primary"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            className="text"
          >
            =
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
